src/frontend/components/ExpandedSidebar/ExpandedSidebar.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./ExpandedSidebar.css";
+
+const ExpandedSidebar = () => {
+  return (
+    <div className="expanded-sidebar">
+      <NavLink
+        to="/"
+        className={({ isActive }) =>
+          isActive
+            ? "expanded-sidebar-link active-link"
+            : "expanded-sidebar-link"
+        }
+      >
+        <span className="material-icons-outlined expanded-sidebar-icon">
+          home
+        </span>
+        <span className="sidebar-link-text">Home</span>
+      </NavLink>
+
+      <NavLink
+        to="/search"
+        className={({ isActive }) =>
+          isActive
+            ? "expanded-sidebar-link active-link"
+            : "expanded-sidebar-link"
+        }
+      >
+        <span className="material-icons-outlined expanded-sidebar-icon">
+          search
+        </span>
+        <span className="sidebar-link-text">Search</span>
+      </NavLink>
+
+      <NavLink
+        to="/explore"
+        className={({ isActive }) =>
+          isActive
+            ? "expanded-sidebar-link active-link"
+            : "expanded-sidebar-link"
+        }
+      >
+        <span
+          title="Explore"
+          className="material-icons-outlined expanded-sidebar-icon"
+        >
+          explore
+        </span>
+        <span className="sidebar-link-text">Explore</span>
+      </NavLink>
+
+      <NavLink
+        to="/history"
+        className={({ isActive }) =>
+          isActive
+            ? "expanded-sidebar-link active-link"
+            : "expanded-sidebar-link"
```

Comment:
The callback inside `className` is being repeated, maybe store it in a variable & pass it over here.

Reply:
Yes. Thanks for suggesting.